x-pack/plugins/security_solution/server/endpoint/lib/artifacts/saved_object_mappings.ts

```diff
@@ -13,80 +13,32 @@ import { migrations } from './migrations';
 export const exceptionsArtifactSavedObjectType = ArtifactConstants.SAVED_OBJECT_TYPE;
 export const manifestSavedObjectType = ManifestConstants.SAVED_OBJECT_TYPE;
 
-export const exceptionsArtifactSavedObjectMappings: SavedObjectsType['mappings'] = {
-  properties: {
-    identifier: {
-      type: 'keyword',
-    },
-    compressionAlgorithm: {
-      type: 'keyword',
-      index: false,
-    },
-    encryptionAlgorithm: {
-      type: 'keyword',
-      index: false,
-    },
-    encodedSha256: {
-      type: 'keyword',
-    },
-    encodedSize: {
-      type: 'long',
-      index: false,
-    },
-    decodedSha256: {
-      type: 'keyword',
-      index: false,
-    },
-    decodedSize: {
-      type: 'long',
-      index: false,
-    },
-    created: {
-      type: 'date',
-      index: false,
-    },
-    body: {
-      type: 'binary',
-    },
-  },
-};
-
 export const manifestSavedObjectMappings: SavedObjectsType['mappings'] = {
+  dynamic: false,
   properties: {
     created: {
       type: 'date',
-      index: false,
```

Comment:
I didn't see an explanation in the comments/PR so I'll ask here: why are we _adding_ mappings for these fields, when they previously weren't needed? These mappings can be added at any point in the future, but this is our last opportunity to remove anything unnecessarily indexed (or specified to not be indexed).

Reply:
@rylnd apologies - I was doing some smoke testing locally with the fields removed first.  All looks OK, so I pushed up the changes.